README.md

```diff
@@ -1,8 +1,25 @@
 # exlibris
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md) 
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
 
 `exlibris` is a social network dedicated to tracking and discussing what you're reading.
 
 This will be filled out more once we have a more fleshed out project.
 
-It was created during the 2020 employee hackathon at [ACV Auctions](https://acvauctions.com) and is being actively developed by its maintaners afterward.
+## Developing
+exlibris is written in [Vue.js](https://vuejs.org/) and [go](https://golang.org/), with ActivityPub support coming from [go-fed](https://github.com/go-fed/activity). It uses [postgreSQL](https://postgresql.org/) as a database, though you don't need it installed. [Docker](https://www.docker.com/) is required to build locally.
+
+### Environment files
+You need to provide environment variables for the database, app (Vue) and api (go). Copy any file ending in `.env.dist` to `.env` (ie `app.env.dist` to `app.env`) and fill in as necessary.
+
+### Hot reload
+To start a development server:
+
+```bash
+make run-local
```

Comment:
I'd probably suggest make dev over make run-local, which will build a containerized development environment so you don't need anything but docker.

run-local is the "you should know what you're doing" sort of environment.